client/src/index.jsx

```diff
@@ -1,15 +1,37 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import $ from 'jquery';
+import Bonuses from './components/bonuses.jsx';
+
 
 class App extends React.Component {
   constructor(props) {
     super(props);
+    this.state = {
+      bundleNumber: 1, //number
```

Comment:
How are you planning to confirm that this is a number when it isn't hard coded? And where will errors related to this value be handled?

Reply:
I am rendering basic data as text first and then will write everything else